### ISSUE TITLE
Checking permission returns 'true' if permission doesn't exist

### ISSUE THREAD

**Issue by hotsaucejake** [thedevdojo/voyager#1172]
- Laravel Version: 5.4.19
- Voyager Version: 0.11.9
- PHP Version: 7.1
- Database Driver & Version:

### Description:
Created a custom controller for one of my routes.  In testing, I checked to see if it had a permission without actually creating that permission in the DB.  **It always returns 'true' if the permission does not exist.**  However, when I create the permission in the DB and don't assign it to the user then it returns false (as expected).

### Steps To Reproduce:

My ````routes/web.php```` file has this:

````php
Route::get('/home', 'HomeController@index');
````

My ````app/Http/Controllers/HomeController.php```` has the following:

````php
namespace App\Http\Controllers;

use Illuminate\Http\Request;
use TCG\Voyager\Facades\Voyager;

class HomeController extends Controller
{
    /**
     * Create a new controller instance.
     *
     * @return void
     */
    public function __construct()
    {
        $this->middleware('auth');
    }

    /**
     * Show the application dashboard.
     *
     * @return \Illuminate\Http\Response
     */
    public function index()
    {
      dd(Voyager::can('blahblahblah'));
      // return view('home');

    }
}
````

**This will return true.**  However, if I create a permission and don't assign it to any users, it returns false (as expected).  Only when the permission does not exist it returns true.

**Comment by akazorg** [comment]
Thanks for reporting that, this is a serious security issue.

**Comment by marktopper** [comment]
This is fixed in #1285 for `v1.0-pre-release` since this is a breaking change.
